toast-tk-runtime/src/main/java/com/synaptix/toast/runtime/reader/TestScriptProvider.java

```diff
@@ -0,0 +1,10 @@
+package com.synaptix.toast.runtime.reader;
+
+/**
+ * Created by Nicolas Sauvage on 21/06/2016.
+ */
+public class TestScriptProvider {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty class, write its code or make it an "interface". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS2094)